.travis.yml

```diff
@@ -10,8 +10,8 @@ before_install:
   - travis_retry npm install --loglevel=http
   - >
     openssl aes-256-cbc \
-      -K $encrypted_4fb8e306b739_key \
-      -iv $encrypted_4fb8e306b739_iv \
```

Comment:
@psquickitjayant ok I guess we can default to mocks so it works in Travis. To run locally against a live API I think we'd need a place for the customer to input their subdomain and token. Can these be variables the tester configures? I'm not sure where they would put them but perhaps in the grunt configuration?